File: spring-data-requery/src/test/java/org/springframework/data/requery/repository/sample/ProductRepository.java
package org.springframework.data.requery.repository.sample;

import org.springframework.data.requery.domain.sample.Product;
import org.springframework.data.requery.repository.RequeryRepository;

/**
 * ProductRepository
 *
 * @author debop@coupang.com
 * @since 18. 6. 14
 */
public interface ProductRepository extends RequeryRepository<Product, Long> {
}

File: spring-data-requery/src/test/kotlin/org/springframework/data/requery/repository/sample/ProductRepository.kt
package org.springframework.data.requery.repository.sample

import org.springframework.data.requery.domain.sample.Product
import org.springframework.data.requery.repository.RequeryRepository

/**
 * org.springframework.data.requery.repository.sample.ProductRepository
 *
 * @author debop
 */
interface ProductRepository: RequeryRepository<Product, Long> {
}